README.md

```diff
@@ -2,3 +2,178 @@
+
+## Registry Overview
+
+This repository provides three things:
+
+1. Metadata for every package version in the registry, stored in the `metadata` directory
+2. Package sets, which are updated daily and stored in the `package-sets` directory
+3. An API for registering, updating, transferring, and unpublishing packages, as well as for making manual package set updates
+
+The PureScript Registry as a whole consists of this repository plus a few more resources:
+
+1. The [Registry Index](https://github.com/purescript/registry-index) stores the manifest files for every package version in the registry. Package managers use the registry index to rapidly determine relationships among packages at specific versions.
+2. The [Registry Storage](https://packages.registry.purescript.org) stores tarballs of packages that have been uploaded to the registry. Package managers download dependencies from the registry storage.
+3. The [Registry Dev](https://github.com/purescript/registry-dev) repository is used to develop the registry. The registry-dev repository is also a library, which package managers can use to easily integrate with the registry by reusing its types and functions.
+
+## Using the Registry API
+
+Most PureScript users will only use the Registry API via their package manager. However, it is also possible to interact with it directly via GitHub issues in this repository. The guide below separates _Operations_ (API calls anyone can make) from _Authenticated Operations_ (API calls that can only be made by package owners or registry trustees).
+
+When you submit an issue with the correct payload, this repository's CI will execute the API. Any errors will be reported as a comment on the issue you opened; you can call the API again by commenting on the same issue you opened with another valid payload. If the operation completes successfully then that will be reported to you in a comment and the registry will close the issue.
+
+### Operations
+
+The Registry API allows anyone to add a package, update a package, or upgrade the package set. To add or update a package, your package must contain a [`purs.json` manifest file](https://github.com/purescript/registry/blob/24db591b992b99730ea23211d746bc4c5fe295d7/src/Registry/Schema.purs#L21-L30). For an example, see the [manifests for the `prelude` package](https://github.com/purescript/registry-index/blob/main/pr/el/prelude).
+
+#### Add a Package
+
+To register a new package, open an issue with the JSON representation of the [Addition](https://github.com/purescript/registry/blob/24db591b992b99730ea23211d746bc4c5fe295d7/src/Registry/Schema.purs#L250-L255) type. The registry will fetch the repository at the provided ref and attempt to register the package, compile it with the compiler version in the `buildPlan` field, upload the documentation to Pursuit, and add the package to the day's package set batch.
+
+```jsonc
+{
```

Comment:
Do we support an issue field that is not unformatted-json?

E.g. users might want to use the backticks (```, etc) to make a json block look nice, I guess we should support that?

Reply:
For now we only support a raw JSON string, but I agree it would be nice to support that. We could trim the string, try to parse and throw away backticks, and then parse the inner JSON.